### PR TITLE
Clean up go runtime testrig

### PR DESCRIPTION
**Pull request by jimidle**
Keep a few handy files in the internal directory of the go runtime, but exclude the generated things from github.

This makes it trivial to drop in a grammar that users may have found causes go problems, and automatically work on the latest version of the go runtime and tool.

### CODE REVIEW

**Comment by jimidle** [comment]
Seems the Github triggers failed onthe check. No reason not to merge this I think @parrt

File: runtime/Go/antlr/internal/testrig/README.adoc [hunk 1]
```diff
@@ -1,3 +1,12 @@
-# Test rig for go
+= Test rig for go
 
-This test rig will build and run the grammar file test.g4 with the `input` file and turn on all tracing options.
\ No newline at end of file
+This test rig will build and run the grammar file test.g4 with the `input` file for local testing by maintainers.
+
+You should copy the `dev` brnach version of the antlr4 complete jar in to the antlr directory and you can then run
```
Comment (KvanTTT) [inline runtime/Go/antlr/internal/testrig/README.adoc:5]: Please fix typo:

```suggestion
You should copy the `dev` branch version of the antlr4 complete jar in to the antlr directory and you can then run
```

File: runtime/Go/antlr/internal/testrig/test/.keep
Comment (KvanTTT) [inline runtime/Go/antlr/internal/testrig/test/.keep:1]: Is this file really needed? Can't this directory be created by test runner?

**Review by KvanTTT — CHANGES_REQUESTED** [review]

File: runtime/Go/antlr/internal/testrig/README.adoc [hunk 2]
```diff
@@ -1,3 +1,12 @@
-# Test rig for go
+= Test rig for go
```
[the remark this answers is not in the document]
Comment (parrt) [reply runtime/Go/antlr/internal/testrig/README.adoc:1]: I’ve never heard of that as a header section. Let’s keep it consistent I guess.

**Comment by jimidle** [comment]
This has nothing to do with the normal test rig guys. This is just an
internal part of the go code. I guess I should have called it go test kit
or something  like that. Nobody but me sees this.

Adoc uses = for headers, markdown uses #. Audi, it’s just a placeholder
until I do a doc PR, hopefully tomorrow

.keep is because there is no script. This isn’t part of the runtime tests.
It’s just there to make the directory stay put. Normal practice.

I’ll fix the typo of course and check the .gitignore (but it doesn’t hurt
to definitely specify it in that directory as so sone might change it
elsewhere.

Please look again at which files I  am changing here. It’s nothing to do
with the test kit, it’s just an internal test area for go only.

File: runtime/Go/antlr/internal/testrig/test/.keep
Comment (jimidle) [reply runtime/Go/antlr/internal/testrig/test/.keep:1]: I have removed this internal directory to avoid confusion - I will keep it elsewhere.

File: runtime/Go/antlr/internal/testrig/README.adoc (hunk 1 above)
Comment (jimidle) [reply runtime/Go/antlr/internal/testrig/README.adoc]: I have removed this internal directory to avoid confusion - I will keep it elsewhere.

File: runtime/Go/antlr/internal/testrig/README.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (jimidle) [reply runtime/Go/antlr/internal/testrig/README.adoc]: This is .adoc, not .md

I have removed this internal directory to avoid confusion - I will keep it elsewhere.

File: .gitignore [hunk 3]
```diff
@@ -135,4 +135,7 @@ runtime/Cpp/runtime/libantlr4-runtime.dylib
 /runtime/Cpp/runtime/libantlr4-runtime.4.12.0.dylib
 
 # Go test and performance trace files
-**/*.pprof
\ No newline at end of file
+**/*.pprof
+runtime/Go/antlr/internal/testrig/antlr/*.jar
```
[the remark this answers is not in the document]
Comment (jimidle) [reply .gitignore:139]: Over specifying is a thing in .gitignore. If someone changed the global ignore for their own purposes, then this would still stay in force.

I have removed this internal directory to avoid confusion - I will keep it elsewhere.

**Comment by jimidle** [comment]
This should be good to go now fellas. I am going to create a separate repo in my own space to have a test rig for go runtime grammar testing that is separate from the go runtime source.
